Servers/Administration/OJS.Servers.Administration/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "Http": {
+    "MaxRequestSizeLimit": 35000000
+  },
```

Comment:
Can you put those settings in the bottom, next to AllowedHosts/CORS. Let's keep connection string at the top, as it is changed most frequently.

Reply:
Fixed